src/Validation/Validation.php

```diff
@@ -521,9 +521,11 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * To check if a value is set specifically to false or True see `inList()`.
```

Comment:
@inoas Adding a new option to `boolean()` would be better, as `inList()` seems not suitable to check `boolean`.